From 97e7760a2127232e5663132b1cd1398671c6622e Mon Sep 17 00:00:00 2001
From: "sheng.liu" <sheng.liu@amlogic.com>
Date: Thu, 17 Nov 2022 14:42:23 +0800
Subject: [PATCH] gstreamer1.0-plugins-good: CB1 add case 18 for ARIB_STD_B67
 [1/1]

PD#SWPL-93118

Problem:
no case to handle the value of 18 at id VIDEOTRANSFERCHARACTERISTICS
which will make hdr info can not be showed

Solution:
add the case 18

Verify:
AH212
---
 gst/matroska/matroska-demux.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gst/matroska/matroska-demux.c b/gst/matroska/matroska-demux.c
index bf8ccae..c1f07ae 100644
--- a/gst/matroska/matroska-demux.c
+++ b/gst/matroska/matroska-demux.c
@@ -682,6 +682,9 @@ gst_matroska_demux_parse_colour (GstMatroskaDemux * demux, GstEbmlRead * ebml,
           case 16:
             colorimetry.transfer = GST_VIDEO_TRANSFER_SMPTE_ST_2084;
             break;
+          case 18:
+            colorimetry.transfer = GST_VIDEO_TRANSFER_ARIB_STD_B67;
+            break;
           default:
             GST_FIXME_OBJECT (demux,
                 "Unsupported color transfer characteristics  %"
-- 
2.25.1

